src/explore-education-statistics-common/src/components/Tabs.tsx

```diff
@@ -66,6 +67,9 @@ const Tabs = ({ children, id }: Props) => {
                 onClick={event => {
                   event.preventDefault();
                   setSelectedTab(index);
+                  if (typeof onToggle === 'function') {
```

Comment:
Using the correct type for the `onToggle` prop will actually mean you don't _really_ need to do this check as you already have compile time guarantees that `onToggle` will be a function. It doesn't do any harm to do a `typeof` check any, so you can leave it like this though.